pkg/runtime/cli_test.go

```diff
@@ -44,3 +52,18 @@ func TestParsePlacementAddr(t *testing.T) {
 		})
 	}
 }
+
+func TestAppFlag(t *testing.T) {
+	runtime, err := FromFlags([]string{"--app-id", testAppID, "--app-port", testAppPort, "--app-protocol", testAppProtocol})
+	assert.NoError(t, err)
+	assert.EqualValues(t, testAppID, runtime.runtimeConfig.ID)
+	assert.EqualValues(t, 80, runtime.runtimeConfig.ApplicationPort)
+	assert.EqualValues(t, testAppProtocol, runtime.runtimeConfig.ApplicationProtocol)
+}
+
+func TestStandaloneGlobalConfig(t *testing.T) {
+	runtime, err := FromFlags([]string{"--app-id", testAppID, "--mode", string(modes.StandaloneMode), "--config", "../config/testdata/metric_disabled.yaml"})
+	assert.NoError(t, err)
+	assert.EqualValues(t, testAppID, runtime.runtimeConfig.ID)
```

Comment:
Please check all args.

Reply:
Done. notes: `metrics-port` parsed as a local variable, so could not be checked.